### ISSUE TITLE
Fix LaTeX Warnings

### ISSUE THREAD

**Issue by wuan** [LeastAuthority/moonmath-manual#39]
We get the following warnings in the document's final build:

```
Label `Zn' multiply defined.
Label `Zn@cref' multiply defined.
Label `def:Pedersen_hash' multiply defined.
Label `def:Pedersen_hash@cref' multiply defined.
Label `def:chord-and-tangent' multiply defined.
Label `def:chord-and-tangent@cref' multiply defined.
Command \textbackslash invalid in math mode on input line 275.
Command \textbackslash invalid in math mode on input line 278.
Command \textbackslash invalid in math mode on input line 292.
Command \textbackslash invalid in math mode on input line 880.
Reference `def:hilbert_class_domain' on page 115 undefined on in
Characters dropped after `\end{comment}' on input line 621.
Reference `BLS6-weil-pairing' on page 195 undefined on input lin
```

These should be fixed.

**Comment by dd23** [comment]
I addressed almost all of these issues in #45 
The only remaining warning is: Characters dropped after `\end{comment}' on input line 621.

**Comment by wuan** [comment]
Thank you!
